simd/unit_tests/include/TestSIMD_ShiftOps.hpp

```diff
@@ -131,8 +131,6 @@ inline void host_check_shift_ops() {
         for (std::size_t i = 0; i < width; ++i) test_vals[i] *= -1;
         host_check_shift_op_all_loaders<Abi>(shift_right(), test_vals, shift_by,
                                              num_cases);
-        host_check_shift_op_all_loaders<Abi>(shift_left(), test_vals, shift_by,
-                                             num_cases);
```

Comment:
What is up with this?

Reply:
Left-shifting negative values is undefined behavior.

Reply:
yeah makes sense just looked it up myself.